service/generator/generator.go

```diff
@@ -0,0 +1,342 @@
+package generator
```

Comment:
I think we want to move it to `/pkg` as it will be also used in `/cmd`.

Reply:
Moved to `/pkg`.